### PR TITLE
Adding README for AI Library

### LINKED ISSUE #30: Add README for AI Library

### PR DESCRIPTION
**Pull request by crobby**
Closes #30

### CODE REVIEW

**Comment by LaVLaS** [comment]
LGTM

Do we need to add issues to track adding links to dependent resources (Seldon) or projects (ai-library) when those component are added?

File: ai-library/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+There are 2 folders that contain installation files:
+
+`cluster/base` This folder includes the yaml files that will install the AI Library CRD on your cluster.
+
+`operator/base` This folder includes the yaml files to set up the deployment for the operator in your namespace along with the service account, role, and role binding that is necessary.
+
+### Installation
+To install AI Library add the following to the `kfctl` yaml file.
+
+Note, to use AI Library, you also need to have Seldon installed.
```
Comment (vpavlin) [inline ai-library/README.md:15]: Should this have a link to Seldon README and maybe mention that Seldon has to come before AI Library?

Comment (crobby) [reply ai-library/README.md:15]: Happy to add that.

**Pull request merged**